pkg/gen/nodejs-templates/yaml.ts.mustache

```diff
@@ -87,7 +87,7 @@ import * as outputs from "../types/output";
 
     export interface ConfigOpts {
         /** JavaScript objects representing Kubernetes resources. */
-        objs: any[];
+        objs: pulumi.Input<pulumi.Input<any>[]>;
```

Comment:
What necessitated this change?

Reply:
IIRC, the previous declaration wasn't type checking with the revised logic in `parse`

Reply:
Can you point out the file/line?

Reply:
https://github.com/pulumi/pulumi-kubernetes/blob/39234020502e2ec02bd2ffeac5d39dd2570d67e1/sdk/nodejs/yaml/yaml.ts#L2461
because `yamlLoadAll` returns a `Promise<any[]>`

Reply:
Perfect; thanks